### DISCUSSION TITLE
Organizations and repositories access

### DISCUSSION THREAD

**Discussion by rgbsys** [community/community#61990] [New to GitHub]
I recently upgraded an organization to the Team plan and created 6 teams with 12 users. 

I took one of my private repositories, removed all the users that had access and moved it inside the organization giving access to the teams I created. 

The problem is that one of the users who had access before moving the repository into the organization was able to commit and push it after moving. This user is not part of the 12 users in this organization.

Where can I check how this user is to do these operations?

**Comment by queenofcorgis** [comment]
Hey there - I'm looking into this for you - thanks for your patience!

**Comment by queenofcorgis** [comment]
Can you look at your repository's settings (under owner) in the access section to see if they are showing up as an outsider collaborator?

**Comment by rgbsys** [comment]
Access section do you mean "collaborators and teams" ? If so, there are only the teams that I registered to have access. All team users are members

More info. we have 13 seats, all taken. I think to add an outsider I would need to buy one more license, right?

![image](https://github.com/community/community/assets/60619721/db75b223-6fe3-4978-aab1-b22b0232c267)

**Reply by queenofcorgis** [reply]
Thanks for sharing - would you mind sharing the username of the person who has access but shouldn't please so I could take a further look?

**Reply by rgbsys** [reply]
The user who made the commit is https://github.com/PHtorres. He had access before the repository was migrated into the organization.

![image](https://github.com/community/community/assets/60619721/17879e90-f1d8-4e1a-80ee-ef830bf83964)

**Reply by queenofcorgis** [reply]
Thank you - do you have access to the audit logs for the repo(s)/org in question ?

**Reply by rgbsys** [reply]
Yes

**Reply by queenofcorgis** [reply]
Are you able to check their access in those? That might tell more of the story as on my end they're still showing up as a collaborator on two of your repos.

**Reply by rgbsys** [reply]
Yes I can access. 

He has access to two repositories: rgbsys/my-site-receivables and rgbsys/helpers-sql. These repositories were not migrated to the organization rgbtec and the repository he committed and should not is rgbtec/mobicredito-app

All repositories were under the rgbsys account and he had access to some, including mobicredito-app. This repository was moved into the rgbtec organization and even without access he was able to commit

How can I do this search in the audit-log?

**Reply by queenofcorgis** [reply]
Hi there - our docs detail this - check this [one out](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization).

**Reply by rgbsys** [reply]
I didn't find anything that justifies the user having been able to commit.

There is nothing logged about giving access to this user.

**Reply by rgbsys** [reply]
More information. 

He used a Mac here at the company via Anydesk. He uses the default mac account, I logged into the mac and just create a new branch, deleted a file, did commit and push with his user.

How can this be possible?

![image](https://github.com/community/community/assets/60619721/530de0cf-4482-49c2-91cb-9355939def91)

**Reply by queenofcorgis** [reply]
Hi @rgbsys - I appreciate all the investigating you've been doing, but given the nature of this your next step is opening a ticket on our [Support page](https://support.github.com/contact). Please let me know once this has been opened so I can make sure the team has all the information we've already gathered.

**Reply by rgbsys** [reply]
Thanks for your help. I opened the ticket https://support.github.com/ticket/personal/0/2284159

**Reply by queenofcorgis** [reply]
Wonderful - thank you again for all the back and forth.

**Comment by ghost** [comment]
This is Test Answer.
